### DISCUSSION TITLE
getting table data

### DISCUSSION THREAD

**Discussion by kamurj** [AutomaApp/automa#1813] [Q&A]
Hi everyone. I would appreciate if someone can help me see what I am doing wrong.

I have a table 'p_amz' with columns p_title, p_url etc.
I can store data properly, but can't use that data in forms block.

If I want to access row one values for p_title for example, how i should refer to it within the value field of forms block? like this {{p_amz.1.p_title}}, {{p_amz.[1].[p_title]}}... none of these seem to work. 

Thank you.

**Comment by kamurj** [comment]
another thing I need with is accessing arrays in forms block. Using push i can assign values to p_title array but then when I want to use the forms block to copy array values to html forms it does not work. I tried {{variables.p_title.[0]}}, {{variables.p_title.0}}, variables.p_title.[0] and none seem to work.

**Reply by guyong613** [reply]
key point , read table data have NO MATTER WITH  TABLE name,
for you ,you can use this syntax {{table.1.p_title}}, NOT {{p_amz.1.p_title}}.
https://docs.automa.site/workflow/expressions.html
